css/style.css

```diff
@@ -52,6 +52,8 @@ header a:hover {
 
 footer {
   text-align: center;
+  padding-top: 600px
+  border: 1px solid red;
```

Comment:
Can you make these changes just in my-board.css? We won't want these changes on the landing page or the about page.... at least for the padding but probably the border too.

Reply:
agreed